app/assets/javascripts/hyrax/app.js.erb

```diff
@@ -91,7 +91,7 @@ Hyrax = {
         // Do not create a consumer if user is not logged in
         if ($("meta[name='current-user']").length === 0)
             return;
-        var consumer = ActionCable.createConsumer("<%= Hyrax::Engine.routes.url_helpers.notifications_endpoint_path %>");
+        var consumer = ActionCable.createConsumer("<%= Hyrax::Engine.routes.url_helpers.notifications_endpoint_path if Hyrax.config.realtime_notifications? %>");
```

Comment:
This code is never interpreted when realtime notifications are off, because the function returns on line 93.